onnx_xla/operator_registry.cc

```diff
@@ -51,6 +51,23 @@ XlaComputation OperatorRegistry::max(PrimitiveType dataType) {
   return builder.Build().ConsumeValueOrDie();
 }
 
+std::vector<int64> OperatorRegistry::parseOnnxInputSizes(const Node& n,
+                                                         size_t inputIndex) {
+  std::vector<int64> shapeInts;
+  const auto& shapeDims = n.inputs().at(inputIndex)->sizes();
+  if (shapeDims.size() == 0) {  // TODO: Enforce
+    throw std::runtime_error("Missing shape");
```

Comment:
This is probably not ok, Scalar values in ONNX can have empty sizes which is valid

Reply:
how do I know the difference between sizes not begin set as you pointed out here https://github.com/varunjain99/onnx-xla/pull/16 and scalars

Reply:
Chatted offline, @varunjain99 will submit a PR in onnx repo to add a flag in onnx in memory IR.